lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -1454,6 +1454,27 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     Value mult = b.create<arith::MulFOp>(loc, self, alpha);
     return b.create<arith::SubFOp>(loc, other, mult);
   }
+  if (auto atenToDtype = dyn_cast<AtenToDtypeOp>(op)) {
+    Value input = payloadArgs[0];
+    Type inType = input.getType();
+    Type outType = atenToDtype.getType().cast<ValueTensorType>().getDtype();
+    Value result;
+    if (!inType.isF32()) {
+      atenToDtype.emitError("unimplemented: non-floating point dtype");
+      return nullptr;
+    } else {
+      if (inType == outType)
```

Comment:
Just a note that I will make some changes to `promoteScalarToDtype` so that it can be used here. For now, it's ok to leave the code like this. I will clean it up as part of my PR.